src/components/SQLResults/SQLResultsTable.tsx

```diff
@@ -268,70 +272,74 @@ function SQLResultsTable({ result }: Params) {
     return renderErrorTable(result);
   }
 
-  let columns;
-  let data: object[];
+  let output:
+    | DataTableColumnData<JSONData>
+    | DataTableColumnData<CSVData>
+    | DataTableColumnData<TableViewData>
+    | DataTableColumnData<NoColumnsData>;
 
   if (tableResultsFormat == 'json') {
```

Comment:
There are a lot of `==` conditions in this component... Since it can cause bugs (not in this case specifically), should we update them to `===`?